_examples/acl/main.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 Fastly, Inc.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/fastly/compute-sdk-go/acl"
+	"github.com/fastly/compute-sdk-go/fsthttp"
+)
+
+func main() {
+	fsthttp.ServeFunc(func(ctx context.Context, w fsthttp.ResponseWriter, r *fsthttp.Request) {
+		aclh, err := acl.Open("example")
```

Comment:
Understanding this is an example, should the `err` be handled?

Reply:
Oops, thanks.

Reply:
Fixed and force-pushed.